### PR TITLE
Adding a 'replace_emoticon' flag

### PR DESCRIPTION
**Pull request by realdeveloperongithub**
Adding a 'replace_emoticon' flag to provide ways to disable emoticon replacing

### CODE REVIEW

File: efb_wechat_slave/utils.py [hunk 1]
```diff
@@ -150,7 +151,7 @@ def __call__(self, flag_key: str) -> Any:
         return self.config[flag_key]
 
 
-def wechat_string_unescape(content: str) -> str:
+def wechat_string_unescape(content: str, replace_emoticon: bool = True) -> str:
```
Comment (blueset) [inline efb_wechat_slave/utils.py:154]: Look like no references of `wechat_string_unescape` is updated to pass in the new parameter.

**Review by blueset — CHANGES_REQUESTED** [review]
Left a comment in the code.

Also, can you add your change to `CHANGES.rst`?

File: efb_wechat_slave/utils.py (hunk 1 above)
Comment (realdeveloperongithub) [reply efb_wechat_slave/utils.py:154]: updated slave_message.py and CHANGES.rst

**Comment by realdeveloperongithub** [comment]
updated slave_message.py and CHANGES.rst

**Review by blueset — APPROVED** [review]

**Comment by blueset** [comment]
Thank you for the awesome contribution!

**Pull request merged**